### DISCUSSION TITLE
Citations docs are too new! Or, how to avoid updating docs until a release?

### DISCUSSION THREAD

**Discussion by bryanwweber** [executablebooks/meta#246] [Q&A] [answered]
Hi everyone, I'm using the stable version of JB from PyPI (0.9.1), which is pinned to sphinxcontrib-bibtex 1.0.0. A couple of days ago, the citation page in the docs (https://jupyterbook.org/content/citations.html#citations-and-bibliographies) was updated for sphinxcontrib-bibtex 2.1.0. When I tried to create a bibliography today, it wasn't working at all until I realized that the page had been updated for a new version of sphinxcontrib-bibtex that I wasn't using.

The question I have is, is there any way to avoid publishing updates to the documentation until the relevant changes are released to PyPI? Thanks, any insight would be much appreciated!

**Comment by chrisjsewell** [comment, marked as answer]
There is a way to do this with readthedocs, but unfortunately this is not just yet supporting jupyter-book. 

In general though we aim to not merge such documentation changes until just before a release. This was simply a mistake by @choldgraf (https://github.com/executablebooks/jupyter-book/commit/e0a41fd592538dac719d6b906eaa73328c7a52dc) 👀 that I've already warned him about before 😜 (https://github.com/executablebooks/jupyter-book/pull/1189#issuecomment-768790429)

**Reply by chrisjsewell** [reply]
To note, there will anyhow be a new release in the next day or two

**Reply by bryanwweber** [reply]
Thanks @chrisjsewell! Glad to know this is on your minds. What's the incompatibility with readthedocs? Can I help fix it?

I saw that 0.10.0 was released, :+1: thank you!
